actions/debootstrap_action.go

```diff
@@ -125,8 +170,7 @@ func (d *DebootstrapAction) Run(context *debos.DebosContext) error {
 	if !d.CheckGpg {
 		cmdline = append(cmdline, fmt.Sprintf("--no-check-gpg"))
 	} else if d.KeyringFile != "" {
-		path := debos.CleanPathAt(d.KeyringFile, context.RecipeDir)
-		cmdline = append(cmdline, fmt.Sprintf("--keyring=%s", path))
+		cmdline = append(cmdline, fmt.Sprintf("--keyring=%s", d.KeyringFile))
```

Comment:
what if spaces are in `d.KeyringFile` ?

Reply:
this also applies to the other arguments

Reply:
I answered in [comment for #230](https://github.com/go-debos/debos/pull/230#discussion_r484981271): since we are using `os/exec` -- arguments are passed to debootsrap as is. No issues with Verify part.

for different parameters we have an interesting issues related to blank spaces:
- for keyring -- in case if any **directory in the path** to keyring contains blank spaces -- `fakemachine` hangs after mounting such volume, no issues if the keyfile itself have blank spaces.
- for certificate and private-key -- the same story with the path to file -- `fakemachine` hangs. And with blank spaces in file name -- `debootsrap` it doesn't work since it do not expect blank spaces. For example: `PRIVATEKEY="--private-key=$2"` <- this is the argument passed to wget as is and this breaks the work of `debootstrap` itself.

Reply:
TBH I don't think that's the issue for the current MR -- but there are a couple of issues which should be fixed in `fakemachine` and `debootstrap`.

Reply:
Right -- `fakemachine` stuck in `m.RunInMachineWithArgs()`

Reply:
Generally i wouldn't expect debos to be safe against paths with spaces (or with weird characters) especially since we indeed callout to debootstrap and other tools which can break as well; So for this MR i don't see it as a blocker; It might be nice at some point to have tests with spaces in various paths for various actions just to improve debos' correctness here